vibrato/src/dictionary.rs

```diff
@@ -24,6 +24,19 @@ pub use crate::dictionary::word_idx::WordIdx;
 
 pub(crate) use crate::dictionary::lexicon::WordParam;
 
+const VERSION: &str = env!("CARGO_PKG_VERSION");
```

Comment:
I think it is not a good idea to use `CARGO_PKG_VERSION` because the model will need to be updated each time the revision number changes even if the API is not changed.